### PR TITLE
Submission of connectorio membership profile.

### PR DESCRIPTION
**Pull request by splatch**

### CODE REVIEW

File: members/2024-01-07-connectorio.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+address1: 'ConnectorIO sp. z o.o.'
+address2: 'Nieszawska 4B'
+city: '03-382 Warsaw'
+country: 'Poland'
+email: contact@connectorio.com
+web: http://connectorio.com/oh
+twitter: connectorio
+
+---
+ConnectorIO® is service and technology provider focused on openHAB project. We completed number of deployments in Europe - Germany, Switzerland, Liechtenstein and Poland. Our firm is largest publisher of third party extensions for openHAB project. Addons which we support integrate variety of industrial and commercial real estate equipment.
```
Comment (kaikreuzer) [inline members/2024-01-07-connectorio.md:16]:
```suggestion
ConnectorIO® is a service and technology provider focused on openHAB project.
We have completed a number of deployments in Europe - Germany, Switzerland, Liechtenstein and Poland.
Our firm is the largest publisher of third party extensions for the openHAB project.
Add-ons which we support integrate a variety of industrial and commercial real estate equipment.
```

File: members/2024-01-07-connectorio.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+<!-- more -->
+
+ConnectorIO's focuses on safety of software and software integration towards production use.
```
Comment (kaikreuzer) [inline members/2024-01-07-connectorio.md:20]:
```suggestion
ConnectorIO focuses on safety of software and software integration towards production use.
```

File: members/2024-01-07-connectorio.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Over course of years we developed own [cloud connector](https://connectorio.com/services/cloud/connectorio-cloud/), which works independently of openHAB cloud as well as [energy monitoring solution](https://connectorio.com/solutions/energy-monitoring/) suited for resource constrained devices.
```
Comment (kaikreuzer) [inline members/2024-01-07-connectorio.md:21]:
```suggestion
Over the course of years we developed our own [cloud connector](https://connectorio.com/services/cloud/connectorio-cloud/), which works independently of openHAB cloud as well as an [energy monitoring solution](https://connectorio.com/solutions/energy-monitoring/) suited for resource constrained devices.
```

File: members/2024-01-07-connectorio.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+We provide failsafe and hassle free upgrade mechanisms thanks to customized software assemblies and embedded linux distribution based on Yocto project.
```
Comment (kaikreuzer) [inline members/2024-01-07-connectorio.md:22]:
```suggestion
We provide a failsafe and hassle free upgrade mechanisms thanks to customized software assemblies and an embedded Linux distribution based on Yocto project.
```

File: members/2024-01-07-connectorio.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+Our open source integrations work with BACnet, CAN bus, CANopen, OPC UA, Wireless M-Bus, M-Bus as well as PLC equipment manufactured by Beckhoff, Siemens and Fatek using their propertiary protocols.
```
Comment (kaikreuzer) [inline members/2024-01-07-connectorio.md:24]:
```suggestion
Our open source integrations work with BACnet, CAN bus, CANopen, OPC UA, Wireless M-Bus, M-Bus as well as with PLC equipment manufactured by Beckhoff, Siemens and Fatek using their proprietary protocols.
```

File: members/2024-01-07-connectorio.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+**ConnectorIO is registered trademark of Code-House, Łukasz Dywicki**
```
Comment (kaikreuzer) [inline members/2024-01-07-connectorio.md:26]:
```suggestion
**ConnectorIO is a registered trademark of Code-House, Łukasz Dywicki**
```

**Review by kaikreuzer — CHANGES_REQUESTED** [review]
Thanks for the memberpage. I have some suggestions for improved formulations, see below.

**Comment by splatch** [comment]
I've committed and squashed all your suggestions, its ready for next review.

**Review by kaikreuzer — APPROVED** [review]
Thanks!

**Pull request merged**